rust/src/modularity.rs

```diff
@@ -24,15 +24,15 @@ enum Opt {
     Uninstall(InstallOpts),
 }
 
-#[derive(Debug, StructOpt)]
+#[derive(Debug, Parser)]
 struct InstallOpts {
-    #[structopt(parse(from_str))]
+    #[clap(action(ArgAction::Append))]
```

Comment:
I think `#[clap(long)]` is sufficient here, the default is append right?  Although now I'm uncertain what the structopt `parse(from_str)` was doing there.

Reply:
I just tested, it definitely works as expected to use `#[clap(long)]` here.

Reply:
To be clear I am OK to merge as is, we can tweak this in a followup patch.

Reply:
But to elaborate, what I found extremely compelling about the original `structopt` is that it simply "Did The Right And Obvious Thing" when passed a rust struct, in the same way as serde, generally.

I mean, almost any other behavior for `Vec<String>` would be just bizarre and wrong.